site/content/en/docs/manual/advanced/cli.md

```diff
@@ -21,9 +22,19 @@ Overview of functionality:
 - Export and download a whole task
 - Import a task
 
-**Usage**
+## Usage
+
+If you deployed CVAT using docker, to access the CLI, run:
 
 ```bash
+docker exec -it cvat bash
```

Comment:
These instructions are not useful. Users don't need to go inside the docker container. It is just inconvenient. Instead the user should clone the cvat repo, install necessary dependencies and run cli.py. I hope in the future we will have a pip package for cvat command line line tools like cvat-cli.